.github/workflows/smoke-test.yml

```
@@ -137,7 +114,7 @@ jobs:
           restore-keys: ${{ runner.os }}-cargo-
 
       - name: Test
-        run: cargo test --verbose --features untrusted,ffi,polars
+        run: cargo test --verbose --all-features --lib
```

```suggestion
        run: cargo test --verbose --all-features
```
I'm uncomfortable disabling doctests entirely right now. Maybe hold off on this part?
- Are there particular doctests that are using a lot of time, or is it simply that doctests in general are slow to run? Sometimes just a couple tests dominate.
- If they are dropped from smoke test, could we make sure they are run nightly? Might already be the case.
- Do we want to discourage doctests going forward? Or only use them on high visibility functions? Or make sure that single doctests cover a lot of ground, to make up for the compilation cost?